manager/testdata/e2e/module-template.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright 2020 IBM Corp.
```

Comment:
module-template is not required. I think it was left as a template for how the modules should be written (charts, templates, values,...)